torchvision/models/detection/roi_heads.py

```diff
@@ -479,6 +479,50 @@ def paste_masks_in_image(masks, boxes, img_shape, padding=1):
     return ret
 
 
+def check_target_item(target, key, dtype, shape, shape_string=None):
```

Comment:
Can you prefix this function with an underscore so that it is private? `_check_target_item`

Reply:
Of course.